src/main/java/org/broadinstitute/hellbender/tools/coveragemodel/linalg/FourierLinearOperator.java

```diff
@@ -0,0 +1,99 @@
+package org.broadinstitute.hellbender.tools.coveragemodel.linalg;
```

Comment:
I suspect that we will remove these classes in the future in favor of an FFT-free CNV-avoiding penalty. Let me leave them as is for the time being.